avalancheup-aws/src/apply/mod.rs

```diff
@@ -1776,7 +1779,27 @@ pub async fn execute(log_level: &str, spec_file_path: &str, skip_prompt: bool) -
                                 ),
                             ]))
                             .await
-                            .unwrap();
+                        {
+                            Ok(o) => o,
+                            Err(e) => {
+                                retry_count += 1;
+                                if retry_count == max_retry {
+                                    return Err(io::Error::new(
```

Comment:
Previously, this code panicked, now it returns an error. Are all the callers handling this correctly?

Reply:
I think so yes. There's really only one main caller the CLI command `main` entrypoint. All errors are surfaced back up to the main entrypoint. Currently in a lot of cases the code panics and short circuits, but return an error should be fine.